EmbodiedSuffering_oM/Elements/Slavery.cs

```diff
@@ -31,11 +31,20 @@
 
 namespace BH.oM.EmbodiedSuffering.Elements
 {
-    [Description("An Embodied Suffering object used for defining the prevalence of modern slavery in each country's population. Measured by the number of victims per 1000 people.")]
-    public class Slavery : BHoMObject
+    [Description("An object that estimates the risk of labour exploitation by measuring the number of victims of modern slavery per 1000 people, violations against workers' rights (freedom of association), monitoring worker voice, and other socioeconomic factors that lead to exploitation.")]
+    public class LabourExploitationRisk : BHoMObject
     {
-        [Description("The number of enslaved people per 1000 population.")]
-        public virtual double NumberOfEnslavedPeople { get; set; } = 0;
+        [Description("The prevalence of individuals involved in modern slavery, measured in victims per 1000 population.")]
+        public virtual double VictimsOfModernSlavery { get; set; } = double.NaN;
+
+        [Description("Measurement of workers' rights violations, on a scale from 1 to 6 (6 being the most egregious.) Based on scale provided by the International Trade Union Confederation.")]
+        public virtual int FreedomOfAssociation { get; set; } = 0;
+
```

Comment:
For the below default string values for WorkerVoice and Manufacturer, is it enough feedback to the end user to simply have an empty string. Or might it be more clear to have the default string noted as "No Data Available"? Sometimes an empty string can be ambiguous, implying no data(as we intend) or perhaps something went wrong in the script (i.e. the script is working but this part is not, which we do not intend). But if its simpler to just leave it as an empty string, I am fine with that.

Reply:
@michaelhoehn suggested an empty string, as it was easier to establish in checking methods later down the line, "No Data Available" could be established as a check as well @michaelhoehn I'll leave the decision to you

Reply:
understood